src/frontend/AppContainerWrapper.tsx

```diff
@@ -1,189 +1,105 @@
 import * as React from "react";
 import debug from "debug";
-import {
-  NavigationActions,
-  NavigationContainerComponent,
-  NavigationState,
-} from "react-navigation";
-import AsyncStorage from "@react-native-async-storage/async-storage";
 
-import {
-  EDITING_SCREEN_NAMES,
-  ERROR_STORE_KEY,
-  IS_E2E,
-  NAV_STORE_KEY,
-  NO_PRACTICE_BAR,
-  TEMP_HIDE_PRACTICE_MODE_UI,
-  URI_PREFIX,
-} from "./constants";
+import { IS_E2E, URI_PREFIX } from "./constants";
 // import useProjectInviteListener from "./hooks/useProjectInviteListener";
-import bugsnag from "./lib/logger";
 import { PracticeMode } from "./sharedComponents/PracticeMode";
-import DefaultContainer from "./Navigation/DefaultContainer";
-import OnboardingContainer from "./Navigation/OnboardingContainer";
 import { devExperiments, featureFlagOn } from "./lib/DevExperiments";
-
-const AppContainer = devExperiments.onboarding
-  ? OnboardingContainer
-  : DefaultContainer;
+import {
+  NavigationContainer,
+  InitialState,
+  NavigationState,
+  useNavigationContainerRef,
+} from "@react-navigation/native";
+import { AppStack, AppStackList } from "./Navigation/AppStack";
+// import { Linking } from "react-native";
+import Loading from "./sharedComponents/Loading";
+import {
+  getRouteName,
+  hidePracticeBarForRoute,
+  hidePracticeModeTemporarily,
+  inviteModalDisabledOnRoute,
+  loadSavedNavState,
+  persistNavigationState,
+} from "./Navigation/navigationStateHelperFunctions";
```

Comment:
```suggestion
} from "./Navigation/NavigationStateHelperFunctions";
```

Reply:
Refer to @achou11's [comment](https://github.com/digidem/mapeo-mobile/pull/992#discussion_r914064655)